ui/src/components/VolumeListTable.js

```diff
@@ -536,17 +186,52 @@ const VolumeListTable = (props) => {
   };
 
   return (
-    <VolumeListContainer>
+    <Box height={'100%'}>
       <Table
         columns={columns}
         data={volumeListData}
-        nodeName={nodeName}
-        onClickRow={onClickRow}
-        volumeName={volumeName}
-        theme={theme}
-        isSearchBar={isSearchBar}
-      />
-    </VolumeListContainer>
+        defaultSortingKey={'health'}
+        getRowId={(row) => row.name}
+      >
+        <Box
+          display="flex"
+          justifyContent={'space-between'}
+          pt={'16px'}
+          px={'16px'}
+        >
+          <Table.SearchWithQueryParams
+            displayedName={{
+              singular: 'volume',
+              plural: 'volumes',
+            }}
+          />
+          <Button
+            variant={'primary'}
+            label={intl.formatMessage({ id: 'create_new_volume' })}
+            icon={<i className="fas fa-plus" />}
+            onClick={() => {
+              // depends on if we add node filter
+              if (nodeName) {
+                history.push(`/volumes/createVolume?node=${nodeName}`);
+              } else {
+                history.push('/volumes/createVolume');
+              }
+            }}
+            data-cy="create_volume_button"
+          />
+        </Box>
+        <Table.SingleSelectableContent
+          rowHeight="h40"
+          separationLineVariant="backgroundLevel1"
+          backgroundVariant="backgroundLevel2"
+          selectedId={volumeName}
+          onRowSelected={onClickRow}
+          children={(Rows) => {
+            return <>{Rows}</>;
```

Comment:
I think an empty state is missing here. It should likely take the same content of loki/prom/alert manager volume missing.